cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -85,6 +84,21 @@ spec:
               mountPath: /mount-cache-dir
             - name: ceph-csi-config
               mountPath: /etc/ceph-csi-config/
+        - name: liveness-prometheus
+          image: {{ .LivenessImage }}
```

Comment:
This will be the same as `.CSIPluginImage` now, right?

Reply:
yup your right